File: 0684/sesión15 - consulta/SoftVen/SoftVenModel/VentasDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftVen.Model
{
    public class VentasDTO
    {
        private int? ventasId;
        private TiposDocumentosDTO tipoDocumento;
        private string numDocCliente;
        private string nombreCliente;
        private TiendasDTO tienda;
        private VendedoresDTO vendedor;
        private DateTime? fechaVenta;
        private DateTime? fechaCreacion;
        private double? totalVenta;
        private Estado? estado;

        public VentasDTO()
        {
            this.VentasId = null;
            this.TipoDocumento = null;
            this.NumDocCliente = null;
            this.NombreCliente = null;
            this.Tienda = null;
            this.Vendedor = null;
            this.FechaVenta = null;
            this.FechaCreacion = null;
            this.TotalVenta = null;
            this.Estado = null;
        }

        public VentasDTO(int ventasId, TiposDocumentosDTO tipoDocumento, string numDocCliente, string nombreCliente, TiendasDTO tienda, VendedoresDTO vendedor, DateTime fechaVenta, DateTime fechaCreacion, double totalVenta, Estado estado)
        {
            this.VentasId = ventasId;
            this.TipoDocumento = tipoDocumento;
            this.NumDocCliente = numDocCliente;
            this.NombreCliente = nombreCliente;
            this.Tienda = tienda;
            this.Vendedor = vendedor;
            this.FechaVenta = fechaVenta;
            this.FechaCreacion = fechaCreacion;
            this.TotalVenta = totalVenta;
            this.Estado = estado;
        }

        public int? VentasId { get => ventasId; set => ventasId = value; }
        public TiposDocumentosDTO TipoDocumento { get => tipoDocumento; set => tipoDocumento = value; }
        public string NumDocCliente { get => numDocCliente; set => numDocCliente = value; }
        public string NombreCliente { get => nombreCliente; set => nombreCliente = value; }
        public TiendasDTO Tienda { get => tienda; set => tienda = value; }
        public VendedoresDTO Vendedor { get => vendedor; set => vendedor = value; }
        public DateTime? FechaVenta { get => fechaVenta; set => fechaVenta = value; }
        public DateTime? FechaCreacion { get => fechaCreacion; set => fechaCreacion = value; }
        public double? TotalVenta { get => totalVenta; set => totalVenta = value; }
        public Estado? Estado { get => estado; set => estado = value; }
    }
}

File: 0684/sesión25 - Reportes/Back-End/ServidorRESTFul/SoftVenModel/src/main/java/pe/edu/pucp/softven/model/VentasDTO.java
package pe.edu.pucp.softven.model;

import java.util.Date;

public class VentasDTO {
    private Integer ventasId;
    private TiposDocumentosDTO tipoDocumento;
    private String numDocCliente;
    private String nombreCliente;
    private TiendasDTO tienda;
    private VendedoresDTO vendedor;
    private Date fechaVenta;
    private Date fechaCreacion;
    private Double totalVenta;
    private Estado estado;

    public VentasDTO() {
        this.ventasId = null;
        this.tipoDocumento = null;
        this.numDocCliente = null;
        this.nombreCliente = null;
        this.tienda = null;
        this.vendedor = null;
        this.fechaVenta = null;
        this.fechaCreacion = null;
        this.totalVenta = null;
        this.estado = null;
    }
    
    public VentasDTO(Integer ventasId, TiposDocumentosDTO tipoDocumento, String numDocCliente, String nombreCliente, TiendasDTO tienda, VendedoresDTO vendedor, Date fechaVenta, Date fechaCreacion, Double totalVenta, Estado estado) {
        this.ventasId = ventasId;
        this.tipoDocumento = tipoDocumento;
        this.numDocCliente = numDocCliente;
        this.nombreCliente = nombreCliente;
        this.tienda = tienda;
        this.vendedor = vendedor;
        this.fechaVenta = fechaVenta;
        this.fechaCreacion = fechaCreacion;
        this.totalVenta = totalVenta;
        this.estado = estado;
    }

    public Integer getVentasId() {
        return ventasId;
    }

    public void setVentasId(Integer ventasId) {
        this.ventasId = ventasId;
    }

    public TiposDocumentosDTO getTipoDocumento() {
        return tipoDocumento;
    }

    public void setTipoDocumento(TiposDocumentosDTO tipoDocumento) {
        this.tipoDocumento = tipoDocumento;
    }

    public String getNumDocCliente() {
        return numDocCliente;
    }

    public void setNumDocCliente(String numDocCliente) {
        this.numDocCliente = numDocCliente;
    }

    public String getNombreCliente() {
        return nombreCliente;
    }

    public void setNombreCliente(String nombreCliente) {
        this.nombreCliente = nombreCliente;
    }

    public TiendasDTO getTienda() {
        return tienda;
    }

    public void setTienda(TiendasDTO tienda) {
        this.tienda = tienda;
    }

    public VendedoresDTO getVendedor() {
        return vendedor;
    }

    public void setVendedor(VendedoresDTO vendedor) {
        this.vendedor = vendedor;
    }

    public Date getFechaVenta() {
        return fechaVenta;
    }

    public void setFechaVenta(Date fechaVenta) {
        this.fechaVenta = fechaVenta;
    }

    public Date getFechaCreacion() {
        return fechaCreacion;
    }

    public void setFechaCreacion(Date fechaCreacion) {
        this.fechaCreacion = fechaCreacion;
    }

    public Double getTotalVenta() {
        return totalVenta;
    }

    public void setTotalVenta(Double totalVenta) {
        this.totalVenta = totalVenta;
    }

    public Estado getEstado() {
        return estado;
    }

    public void setEstado(Estado estado) {
        this.estado = estado;
    }
    
    
}
